### PR TITLE
Fix/lock date and voting power

### LINKED ISSUE #232: Fix voting power page to show delegated veMENTO
After having locked veMENTO via the vesting safe and delegating voting power to our Ledgers, the governance UI should show our voting power on https://governance.mento.org/voting-power — but it shows 0 instead.

The address dropdown on the top right shows the correct veMENTO amount so it must be a bug on the voting power page.

### PR DESCRIPTION
**Pull request by denviljclarke**
### Description

Updated the voting power page to show the veMENTO balance for users without a lock.

### Other changes

Display correct lock expiration date (showed one week later previously) 
Change voting power page title to "My voting power" instead of my veMENTO lock 

### Tested

I viewed the voting power page with an account with delegated voting power & no lock

### Related issues

- Fixes #232

### CODE REVIEW

File: src/components/locks-list/locks-list.component.tsx [hunk 1]
```diff
@@ -36,12 +40,14 @@ export const LocksList = () => {
...
-  const parsedExpirationDate = format(lock.expiration, "dd/MM/yyyy");
+  const parsedExpirationDate = hasLock
+    ? format(lock?.expiration, "dd/MM/yyyy")
+    : "-";
 
   return (
     <LockInfo
       unlockedMento={unlockedMento}
-      lockedBalance={lockedBalance}
+      lockedBalance={veMentoBalance.formatted}
```
Comment (chapati23) [inline src/components/locks-list/locks-list.component.tsx:50]: can you add thousand separators to these numbers? think it's as simple as `balance.toLocaleString()`, but not sure what the right place for the transformation is, probably the consuming component

File: src/components/locks-list/locks-list.component.tsx [hunk 2]
```diff
@@ -9,10 +9,14 @@ import {
 } from "../locks-lock-info/lock-info.component";
 import { useLockInfo } from "@/lib/hooks/useLockInfo";
 import { RelockForm } from "./locks-relock-form/relock-form.component";
+import useTokens from "@/lib/contracts/useTokens";
```
Comment (chapati23) [inline src/components/locks-list/locks-list.component.tsx:13]: i'm missing the headline change you suggested, maybe you forgot to push?

**Review by chapati23 — COMMENTED** [review]
looks good, delegated balance shows up for me, but I'm missing the two other changes we discussed on discord, maybe you forgot to push them up?

File: src/components/locks-list/locks-list.component.tsx [hunk 3]
```diff
@@ -9,10 +9,15 @@ import {
...
+import { formatUnits } from "viem";
 
 export const LocksList = () => {
   const { address } = useAccount();
-  const { lock, unlockedMento, lockedBalance, hasLock } = useLockInfo(address);
+  const { lock, unlockedMento, hasLock } = useLockInfo(address);
+  const { veMentoBalance } = useTokens();
+
+  const noVotingPower = veMentoBalance.value < BigInt(0);
```
Comment (RyRy79261) [inline src/components/locks-list/locks-list.component.tsx:20]: Shouldn't this line be ` veMentoBalance.value === BigInt(0);`?

**Review by RyRy79261 — APPROVED** [review]

**Review by chapati23 — APPROVED** [review]
nice work

**Pull request merged**